lib/dfu.js

```diff
@@ -295,5 +299,54 @@ var that = module.exports = {
 		);
 	},
 
+	_udevRulesDir: '/etc/udev/rules.d/',
+
+	_missingDevicePermissions: function (stderr) {
+		return stderr && stderr.indexOf("Cannot open DFU device") >= 0;
```

Comment:
We can show an error message, but we shouldn't try to install the rules again. That won't fix anything.